game/src/core/components/PositionComponent.java

```diff
@@ -62,7 +62,7 @@ public PositionComponent(float x, float y) {
     public PositionComponent() {
 
         if (Game.currentLevel() != null) {
-            position = Game.randomTilePoint(LevelElement.FLOOR);
+            position = Game.randomTilePoint(LevelElement.FLOOR).add(new Point(0.5f, 0.25f));
```

Comment:
dafür konstanten anlegen
+ ggf. sperater pr?
+ ich hab ihn https://github.com/Programmiermethoden/Dungeon/pull/1083 hab ich da auch nochmal rumgespielt (ich glaube du kannst den offset dann einfach in https://github.com/Programmiermethoden/Dungeon/blob/master/game/src/core/systems/PositionSystem.java implementieren)

Reply:
habe es erstmal rausgenommen und würde das später als einen neuen Pr wieder einbauen. Aber eher dann direkt gegen den Master.